### DISCUSSION TITLE
Welcome to encapsulation-list-prototype Discussions!

### DISCUSSION THREAD

**Discussion by bermarte** [bermarte/encapsulation-list-prototype#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bermarte** [comment]
first comment

**Comment by 22count22** [comment]
Hello @bermarte! please create an issue with a to-do list inside using our name for everyone to check what he/she has done.

**Reply by bermarte** [reply]
done

**Comment by tahminarasoli** [comment]
See if you can access [FIGMA](https://www.figma.com/file/3sV8V9uzPiZM1mu84wX2yl/Todo?node-id=0%3A1) as well

**Reply by bermarte** [reply]
It's ok, Tahmina you can also export it to a png and upload it to the repo. This way we will link it to the development strategy.

**Comment by bermarte** [comment]
shield.io > https://www.youtube.com/watch?v=Dl-ekLb4quE&t=175s

**Comment by bermarte** [comment]
**for Tahmina:**

- create an HTML todo (with all the classes to be created later)
- fix the HTML for index.html
- style the todo
---
**for Patrick:**

- create issues inside the project-board (delete todo, add todo, delete all todo's, edit text)
- maybe create labels (the text of the label you get from the user stories: delete todo, add todo, delete all todo's, edit text)
- maybe create the milestones (again, the same text of the user stories)
- create a branch and start to implement the delete todo function

Once we have a todo we can create a class (Tahmina)

**Comment by bermarte** [comment]
**What is left:**

- check if the text is empty
- insert the code for the validator
- make text fields editable 
- delete all the todo's
---
**Extras (not necessary):**

- unit-tests
- logger functions
- JSdoc documentation
- Action to create the documentation

**Reply by tahminarasoli** [reply]
> @bermarte 
>     *** insert the code for the validator** I didn't get this part. what do you mean by this and how can I do this?
>

**Reply by bermarte** [reply]
The validator is the buttons on the top, is what is added every time to the pages. It's a library that Evan wrote. It gives you the possibility to validate the HTML, the CSS and all the rest.

**Reply by bermarte** [reply]
@tahminarasoli See https://bermarte.github.io/js-quiz/ on the top you have a row of buttons.
Log the logs, repository, validate HTML and so on.
